### ISSUE TITLE
using --clean flag on app create has no effect

### ISSUE THREAD

**Issue by websmith** [supnate/rekit#148]
Using the following command to create a "clean" app with no samples/example code.

```
rekit create <app-name> [--sass] [--clean]
```

However the --clean flag doesn't appear to do anything. I still end up with an app that has a feature directory called "examples" with all of the sample code.

*I'm running rekit 2.3.3*

Would love to have this feature supported, but if it isn't, maybe you could remove this flag from the docs?

**Comment by supnate** [comment]
Yeah, it's a feature removed because it's easy to remove examples feature now. Will update the docs. Thanks.

**Comment by exocode** [comment]
Wouldn't it be better to create a new app without any demo code.
And add the demo code by passing an argument like `--demo` or `--example`?

The demo is only interesting for people which are new to rekit. After your first touch with rekit you are always will use a blank project boilerplate.
